test_app/test/test_helper.rb

```diff
@@ -0,0 +1,23 @@
+$LOAD_PATH << File.join(File.dirname(__FILE__), '../lib')
+
+require 'test_app'
+require 'minitest/autorun'
+
+# require spec support files and shared behavior
+Dir[File.expand_path('../{support,shared}/**/*.rb', __FILE__)].each do |file|
+  require file
+end
+
+class TestAppTest < Minitest::Test
+  def self.cover(expression)
+    @expression = expression
+  end
+
+  def self.cover_expression
+    unless @expression
```

Comment:
What about using `@expression or fail ...` instead? The semantics should be the same, but it's less code.